ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -861,6 +861,42 @@
               recipients: projectantrea-dev@googlegroups.com rahulj@vmware.com
           triggers: []
           wrappers: []
+      - 'cloud-{name}-{test_name}':
+          test_name: auto-scaling-agent
+          node: antrea-cloud
+          description: 'This is cloud job of auto scaling agents for antrea.'
+          branches:
+          - '${{ANTREA_GIT_REVISION}}'
+          repo_url: '${{ANTREA_REPO}}'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              # Prevent exposing credentials in the console output by adding set +x
+              # This is to avoid other developers removing this set+x by accident
+              set +x
+              sudo gcloud auth login --cred-file=${CREDENTIAL_PATH}
+              sudo gcloud config set project antrea
+              sudo ./ci/jenkins/scale-agent.sh --kind --jenkins-user ${{JENKINS_USER}} --jenkins-token ${{JENKINS_TOKEN}}
+          triggers:
+          - timed: */15 * * * *
+          publishers:
+          - archive:
+              allow-empty: true
+              case-sensitive: true
+              default-excludes: true
+              fingerprint: false
+              only-if-success: false
+          - email:
+              notify-every-unstable-build: true
+              recipients: projectantrea-dev@googlegroups.com
```

Comment:
Not sure whether `notify-every-unstable-build: true` and `*/15 * * * *` will flood the email group.

Reply:
Yes we can remove it in this job.